### PR TITLE
DSD-454: Style Guide Iconography cleanup

### PR DESCRIPTION
**Pull request by bigfishdesign13**
Fixes JIRA ticket [DSD-454](https://jira.nypl.org/browse/DSD-454)

## This PR does the following:
- simplifies story for Style Guide Iconography

## How has this been tested?

- local build of DS Storybook

### Checklist:

- [n/a] I have updated the Storybook documentation accordingly.
- [x] All new and existing tests passed.

### Front End Review:

- [x] View [the example in Storybook](https://pr658-mucbwxgaxxxylrnr4fd3ub9jd6wyyapb.tugboat.qa/)

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr658-mucbwxgaxxxylrnr4fd3ub9jd6wyyapb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/612d080208cf472837587ab7

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr658-mucbwxgaxxxylrnr4fd3ub9jd6wyyapb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/612d080208cf472837587ab7

File: src/components/StyleGuide/Iconography.stories.mdx [hunk 1]
```diff
@@ -111,33 +108,31 @@ The graphic of the `Icon` component is handled through the `IconNames` and `Logo
...
-      </span>
-      <span style={{ marginRight: "1rem" }}>
-        <Icon decorative size={IconSizes.xxlarge} name="check" />
-      </span>
-      <span style={{ marginRight: "1rem" }}>
-        <Icon decorative size={IconSizes.xxxlarge} name="check" />
-      </span>
-    </div>
-  </Story>
+  <div style={{ display: "flex", alignItems: "center" }}>
```
Comment (EdwinGuzman) [inline src/components/StyleGuide/Iconography.stories.mdx:116]: Is it possible to use the `Stack` component from Chakra for this layout?

**Review by EdwinGuzman — APPROVED** [review]

File: src/components/StyleGuide/Iconography.stories.mdx (hunk 1 above)
Comment (bigfishdesign13) [reply src/components/StyleGuide/Iconography.stories.mdx:116]: Sure.  Let me give it a try.

Comment (bigfishdesign13) [reply src/components/StyleGuide/Iconography.stories.mdx:116]: Actually, I will wait on this for now and wait until Chakra is integrated into `development`.

**Pull request merged**